### PR TITLE
ores, ingots, worldgen, and hemp

### PR DESCRIPTION
**Pull request by OneLemonyBoi**

### CODE REVIEW

File: src/main/java/software/bernie/techarium/Techarium.java [hunk 1]
```diff
@@ -1,8 +1,10 @@
 package software.bernie.techarium;
 
+import blusunrize.immersiveengineering.client.DynamicModelLoader;
```
Comment (agnor99) [inline src/main/java/software/bernie/techarium/Techarium.java:3]: Remove this import. It's unused as far as i see and stops the mod from working, if immersive is not present
```suggestion

```

**Comment by bernie-g** [comment]
Looks good! Make sure you add all the items to their appropriate tags

File: src/generated/resources/assets/techarium/lang/en_us.json [hunk 2]
```diff
@@ -1,13 +1,19 @@
 {
+  "block.techarium.aluminium_ore": "Aluminium Ore",
```
Comment (bernie-g) [inline src/generated/resources/assets/techarium/lang/en_us.json:2]: Did you add these manually...? These should be generated in the LangDataGenerator

File: src/main/java/software/bernie/techarium/integration/immersiveengineering/IEIntegration.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                .fluidIn(new FluidStack(Fluids.WATER, 1000))
+                .maxProgress(1000)
+                .rfPerTick(10)
+                .progressPerTick(1)
+                .output(ChancedItemStackList.of(IEItems.Ingredients.hempFiber))
+                .construct()
+                .addCondition(new ModLoadedCondition(ModIntegrations.getIE().orElseThrow(NullPointerException::new).getModID()))
+                .build(consumer,
+                        new ResourceLocation(Techarium.ModID,
+                                "botarium/ie/" + IEItems.Misc.hempSeeds.getRegistryName().getPath()));
```
Comment (bernie-g) [inline src/main/java/software/bernie/techarium/integration/immersiveengineering/IEIntegration.java:43]:
```suggestion
                                "botarium/immersiveengineering/" + IEItems.Misc.hempSeeds.getRegistryName().getPath()));
```

File: src/main/java/software/bernie/techarium/registry/BlockRegistry.java [hunk 4]
```diff
@@ -58,9 +61,15 @@
     public static final RegistryObject<TileEntityType<TopEnabledOnlySlave>> ARBORETUM_TOP_TILE = TILES.register("arboretum_top", () -> TileEntityType.Builder.of(TopEnabledOnlySlave::new, ARBORETUM_TOP.get())
             .build(null));
 
+    // Ores
+    public static final BlockRegistryObjectGroup<Block, BlockItem, TileEntity> ALUMINIUM_ORE =
```
Comment (bernie-g) [inline src/main/java/software/bernie/techarium/registry/BlockRegistry.java:65]: Can we remove this abstraction and just use RegistryObject<Block> here? This seems like too much

**Review by bernie-g — CHANGES_REQUESTED** [review]

File: src/generated/resources/assets/techarium/lang/en_us.json (hunk 2 above)
Comment (OneLemonyBoi) [reply src/generated/resources/assets/techarium/lang/en_us.json:2]: used datagen but seems to have gotten wiped when i rebased

**Comment by bernie-g** [comment]
Looks good, one last thing: could we add smelting recipes/whatever the new ore machine is?

**Review by bernie-g — APPROVED** [review]

**Pull request merged**